core/test-loader.ts

```diff
@@ -1,6 +1,7 @@
 import { ITestFixture, ITest, ITestCase } from "./_interfaces";
 import { FileRequirer } from "./_core";
 import { TestFixture, METADATA_KEYS } from "./alsatian-core";
+const getFunctionName = require("fn-name");
```

Comment:
We could do, but for such a small library I thought it may be worth having a bit more support around the function name.